pkgs/development/tools/turso-cli/default.nix

```diff
@@ -1,27 +1,35 @@
 {
   lib,
-  buildGoModule,
+  buildGo121Module,
   fetchFromGitHub,
 }:
-buildGoModule rec {
+buildGo121Module rec {
   pname = "turso-cli";
-  version = "0.82.0";
+  version = "0.85.1";
 
   src = fetchFromGitHub {
     owner = "tursodatabase";
     repo = "turso-cli";
     rev = "v${version}";
-    hash = "sha256-JFuD10EhR1/nmYPMnNsR/8PUR5ScvWyS+vhg7ZO5TpI=";
+    hash = "sha256-3JyAvxIKDVifGNJ5uOiIcaCvZ3U8u1fdvT6Xmvksnzc=";
   };
 
-  vendorHash = "sha256-Y/pg8+w6B1YQqaZ5wj8QZxiBHAG0Tf3Zec5WlVyA4eI=";
+  vendorHash = "sha256-OiLb+aLZfTLtxNWBJYyyrFXBGbsGKS5rjP+P6RQ94wE=";
+
+  # Build with production code
+  tags = ["prod"];
+  # Include version for `turso --version` reporting
+  preBuild = ''
+    echo "v${version} (nix build)" > internal/cmd/version.txt
+  '';
```

Comment:
Are we able to use `ldFlags` here?

Either by calling `ldFlags` directly and applying them or using the `preBuild` to do so similar to how it's done in [grype](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/security/grype/default.nix) or [syft](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/admin/syft/default.nix)?

We're probably fine to use just the regular binary distribution version.

Reply:
The production code uses `go:embed` to the content of a file as string in build time, I think it fails if the file is not there.

Reply:
I'll try it out when I get back home

Reply:
Indeed, there are two problems with this.
1. `go:embed` requires the file to be present and
2. The `-X` argument for `ldFlags` cannot change an unexported variable

Reply:
Using the distributed binary would require patching the binary. I prefer to rebuild it from source when the it is available.

Reply:
Ok, happy to go ahead with this for the time being then. Only thing I'd suggest is having the `turso --version` output just either `v${version}` or `${version}` instead of `v${version} (nix build)`

Reply:
Ok, I'll change it to `v${version}`